src/main/java/com/mservicetech/openapi/validation/OpenApiValidator.java

```diff
@@ -63,6 +63,7 @@ public OpenApiValidator() {
             spec = new BufferedReader(new InputStreamReader(in, StandardCharsets.UTF_8)).lines().collect(Collectors.joining("\n"));
             openApiHelper = new OpenApiHelper(spec);
             schemaValidator = new SchemaValidator(openApiHelper.openApi3);
+            in.close();
```

Comment:
we should put this line into the finally block. Or just use the try with the resource.

Reply:
I tried to use try block like this:
try (InputStream in = this.getClass().getClassLoader().getResourceAsStream(OPENAPI_YAML_CONFIG)) {
    if (in == null) {
        in = this.getClass().getClassLoader().getResourceAsStream(OPENAPI_YML_CONFIG);
        if (in == null) {
            throw new IOException("Cannot load OpenAPI spec file");
        }
    }
    spec = new BufferedReader(new InputStreamReader(in, StandardCharsets.UTF_8))
            .lines()
            .collect(Collectors.joining("\n"));
    openApiHelper = new OpenApiHelper(spec);
    schemaValidator = new SchemaValidator(openApiHelper.openApi3);
} catch (Exception e) {
    logger.error("Initialization failed: " + e);
}

But it is error since try block will set the in (inputstream) as final and cannot load it again. 

So I am going to change it to finally block